kmath-core/src/commonMain/kotlin/space/kscience/kmath/operations/numbers.kt

```diff
@@ -110,11 +108,8 @@ public object RealField : ExtendedField<Double>, Norm<Double, Double> {
  */
 @Suppress("EXTENSION_SHADOWED_BY_MEMBER", "OVERRIDE_BY_INLINE", "NOTHING_TO_INLINE")
 public object FloatField : ExtendedField<Float>, Norm<Float, Float> {
-    public override val zero: Float
```

Comment:
Adding backing fields is incorrect since the values are stored in the class's constant pool, and loading the number from the backing field is equivalent to loading it from the constant pool.

Reply:
By the way, I think these properties could be made inline.

Reply:
In such case the bytecode like 
`LDC 0.0f`
will be assuringly inlined to the use site to prevent `getZero()F` method invokation.

Reply:
Saving space on singletons does not make any sense.

Reply:
OK